command/operator_debug.go

```diff
@@ -94,8 +94,8 @@ Debug Options:
     Duration for pprof collection. Defaults to 1s.
 
   -server-id=<server>,<server>
-    Comma separated list of Nomad server names, "leader", or "all" to monitor for logs and include pprof
-    profiles.
+    Comma separated list of Nomad server names to monitor for logs, API outputs, and pprof profiles.
```

Comment:
As long as we're making edits here, can we hard-wrap this message to 80 cols?

Reply:
I took the opportunity to hard-wrap all of the help text at 80 columns, separated Consul/Vault options into sections, and moved the most pertinent info to the bottom to make it more visible.